apps/web/src/app/board/board.module.css

```
@@ -0,0 +1,418 @@
+}
+
+.articleTitleLink:hover {
+  text-decoration: underline;
+}
+
+.articleTitle {
+  margin: 0;
+  font-size: 1rem;
+  font-weight: 700;
+  color: #111827;
+  word-break: break-word;
```

Stylelint (17.5.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)